python/docs/source/reference/pyspark.sql/protobuf.rst

```diff
@@ -0,0 +1,28 @@
+..  Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Let's add `protobuf` at `python/docs/source/reference/pyspark.sql/index.rst`

Reply:
@HyukjinKwon It has already been added above.